images/Dockerfile.ci

```diff
@@ -1,5 +1,15 @@
 # This Dockerfile builds the image used by the e2e-metal-ipi test steps in the OpenShift CI.
 # For more details about the test see https://steps.svc.ci.openshift.org/job/openshift-baremetal-operator-master-e2e-metal-ipi
 
-FROM centos:7
-RUN echo "noop"
\ No newline at end of file
+FROM registry.svc.ci.openshift.org/openshift/release:golang-1.13
+ENV HOME /output
+RUN INSTALL_PKGS="ansible python-pip" && \
```

Comment:
use a dockerfile environment variable instead of embedding INSTALL_PKGS in the RUN instruction

Reply:
What's the reason in this case? It's not going to be used any further

Reply:
best practice, better readability, the resulting dockerfile will be easier to maintain
*if* you need a variable inside a RUN, using ENV or ARG, according to the case, should be the way to go, otherwise let's just use a bash script
Anyway, I'm not going to block the PR just for that